mesonbuild/interpreter/compiler.py

```diff
@@ -636,8 +636,7 @@ def has_header_symbol_method(self, args: T.Tuple[str, str], kwargs: 'HeaderKW')
     def notfound_library(self, libname: str) -> 'dependencies.ExternalLibrary':
         lib = dependencies.ExternalLibrary(libname, None,
                                            self.environment,
-                                           self.compiler.language,
-                                           silent=True)
+                                           self.compiler.language)
```

Comment:
Why was silent deleted here?

Reply:
With this patch, `ExternalLibrary.__init__` no longer uses the `silent` argument, because the message printing code was moved out from it. But I should have just removed the argument `__init__` method altogether. I have fixed that in the new patch version.